CONTRIBUTING.md

```
@@ -1,54 +1,86 @@
-    ```
+## Commit Messages
 
-4. Run checks:
+[Conventional Commits](https://www.conventionalcommits.org/) format is enforced. A scope
+is required:
 
-    ```bash
-    make pre-commit
-    make ci
-    ```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)